### DISCUSSION TITLE
Langsame Verbindung - Konfiguration friert ein

### DISCUSSION THREAD

**Discussion by matthiasjanke** [jomjol/AI-on-the-edge-device#2961] [General]
### The Problem

Hallo zusammen,

ich habe zwei ESP32 Cam Module mit AI on the Edge Release: v15.7.0 installiert, um meinen Gaszähler als auch den Wasserzähler auszulesen.

Das ESP32 Cam Module mit dem folgenden Chip läuft reibungslos:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92435210/9b061c44-8122-43ac-a032-52c2567be184)

Das ESP32 Cam Module mit diesem Chip funktioniert kaum bis gar nicht:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92435210/77146e0c-8c38-466c-85a7-eb6b875be3a7)

Bei dem Modul mit der Chip Revision 301 kann kaum eine Verbindung zum Webserver aufgebaut werden. Wenn eine Verbindung aufgebaut wurde ist die Bedienung ist sehr schwerfällig und friert ein. Eine Konfiguration ist nicht wirklich möglich.

Beide ESP32 Cam Module befinden sich 2m von einander getrennt im gleichen Raum und sind mit gleichen WLAN verbunden. 

Vermutlich liegt es an den unterschiedlichen Chip Revisionen. Gibt es dazu eine Lösung?

### Version

Release: v15.7.0

### Logfile

```shell
-
```

**Comment by pedromfa** [comment]
What about writing in english so everyone else understands? :/

**Comment by friedpa** [comment]
As requested the translation:

Hello everyone,
I have installed two ESP32 Cam modules with AI on the Edge Release: v15.7.0 to read my gas meter as well as the water meter.
The ESP32 Cam Module with the following chip is running smoothly:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/42890695/4a1bbe67-6521-4a9d-bd46-d4e19a0d0d5b)
The ESP32 Cam Module with this chip hardly works at all:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/42890695/252c1760-164d-44ba-a694-cff9e05e59bc)
The module with chip revision 301 can hardly establish a connection to the web server. Once a connection has been established, operation is very sluggish and freezes. Configuration is not really possible.
Both ESP32 Cam modules are located 2 meters apart from each other in the same room and are connected to the same WLAN.
This is probably due to the different chip revisions. Is there a solution to this?

Release: v15.7.0

**Comment by HSGEV** [comment]
I can confirm the problem. Same chip revision, same problem

**Comment by caco3** [comment]
> What about writing in english so everyone else understands? :/

We clearly state that German is also accepted!
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/7b3b2182-b6ee-4391-8022-5856814cce3c)

**Reply by pedromfa** [reply]
Not very clear for a non German speaker. 😁 Anyways, the point was so that we (non speakers) can understand what's going on. Thank you for the great work on this project btw. 😉

**Reply by caco3** [reply]
yes, I also prefer english, but we want to be open to Gwrman speakers with only limitted english skills

**Comment by caco3** [comment]
This issue usually is caused by a weak Wifi connection.
Please check your RSSI (Overview page) and the logs.

**Comment by caco3** [comment]
It also seems that the rev. 3 chips have more issues. Search the discussions for this.
IIRC, it can be worked around by forcing the wifi to use specific channels.

https://github.com/jomjol/AI-on-the-edge-device/discussions/2803
https://github.com/jomjol/AI-on-the-edge-device/discussions/2551#discussioncomment-6607170

**Comment by matthiasjanke** [comment]
@caco3 thank you for the two links.

I have switched my Wifi 2,4 ghz Channel to 1 and its works. This setting solved my Problem. Thank you!

**Reply by MatzeAb** [reply]
Same here, the problem was Channel 6. After moving to Channel 1 it works fast without problems.

**Comment by alero63** [comment]
where can i change the channel?

**Reply by caco3** [reply]
you need to do this on your access point.

**Comment by alero63** [comment]
ah ok. that was fine. i already checked.

**Comment by Nool68** [comment]
If you have an AVM FritzBox in mesh mode, ESP devices often have speed problems or very long response times in conjunction with AVM repeaters as a Mesh WiFi bridge and/or AVM repeaters with dual band 2.4GHz+5GHz.
Since I have been using an AVM repeater with only 2.4GHz support and only as a LAN bridge directly in the same room very close to the ESP devices and this AVM repeater uses a different channel, for example channel 1, than the main network channel 11, all of my 5 ESP devices (ESP8266+ESP32) have been running perfectly.

That's probably why I didn't have any problems with chip revision 301 right away.
See my post here:

https://github.com/jomjol/AI-on-the-edge-device/discussions/1732#discussioncomment-9724254
